src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx

```diff
@@ -142,52 +142,52 @@
     <value>Apply DiagnosticAnalyzer attribute for both '{0}' and '{1}'.</value>
   </data>
   <data name="MissingSymbolKindArgumentToRegisterActionMessage" xml:space="preserve">
-    <value>Specify at least one SymbolKind of interest when registering a symbol analyzer action.</value>
+    <value>Specify at least one SymbolKind of interest when registering a symbol analyzer action</value>
   </data>
   <data name="MissingSyntaxKindArgumentToRegisterActionMessage" xml:space="preserve">
-    <value>Specify at least one SyntaxKind of interest when registering a syntax node analyzer action.</value>
+    <value>Specify at least one SyntaxKind of interest when registering a syntax node analyzer action</value>
   </data>
   <data name="MissingOperationKindArgumentToRegisterActionMessage" xml:space="preserve">
-    <value>Specify at least one OperationKind of interest when registering an operation analyzer action.</value>
+    <value>Specify at least one OperationKind of interest when registering an operation analyzer action</value>
   </data>
   <data name="MissingKindArgumentToRegisterActionTitle" xml:space="preserve">
-    <value>Missing kind argument when registering an analyzer action.</value>
+    <value>Missing kind argument when registering an analyzer actionMissing kind argument when registering an analyzer actionMissing kind argument when registering an analyzer action</value>
```

Comment:
I missed this one and will fix it. There is a problem with the fixer when doing the fix-all solution which seems to be fixing the same entry multiple time by doing a concat rather than replace.